projects/observability/src/shared/dashboard/widgets/topology/topology-widget.model.ts

```diff
@@ -16,6 +16,30 @@ export class TopologyWidgetModel {
   })
   public title?: string;
 
+  @ModelProperty({
+    key: 'showLegend',
+    displayName: 'Show Legend',
+    type: BOOLEAN_PROPERTY.type,
+    required: false
+  })
+  public showLegend?: boolean = true;
+
+  @ModelProperty({
+    key: 'showBrush',
+    displayName: 'Show Brush',
+    type: BOOLEAN_PROPERTY.type,
+    required: false
+  })
+  public showBrush?: boolean = true;
+
+  @ModelProperty({
+    key: 'autoZoomToFit',
+    displayName: 'Auto Zoom To Fit',
+    type: BOOLEAN_PROPERTY.type,
+    required: false
+  })
+  public autoZoomToFit?: boolean = false;
```

Comment:
nit: this is a boolean value, better name would be `shouldAutoZoomToFit`

Reply:
done!!